legion/tests/data/empty-docker/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM alpine:3.5
+FROM ubuntu:18.04@sha256:945039273a7b927869a07b375dc3148de16865de44dec8398672977e050a072e
```

Comment:
Excuse me, why was it switched to ubuntu?

Reply:
Opps, fixed